types/transaction_target.go

```diff
@@ -109,24 +102,6 @@ func (t *TransactionTarget) Bytes() ([]byte, error) {
 		if err = builder.AddField(SessionModuleBytesIndex, moduleBytes); err != nil {
 			return nil, err
 		}
-
```

Comment:
@ZhmakAS `Runtime` is serialized now with a calltable instead of the `RuntimeTag` byte.